### ISSUE TITLE
blip vqa2.0 results are much better than the paper

### ISSUE THREAD

**Issue by essamsleiman** [salesforce/LAVIS#488]
I'm evaluating the finetuned blip model for the vqa task on vqa val set following the given vqav2_eval.yaml hyper-parameters. I've added the contents of the file below as well. The paper is reporting a performance of 78.25 on vqa2.0, while after running the script with the following .yaml file, I'm getting a performance of 83.42. I'm not sure why this is the case? I even tried manually fine-tuning blip on vqa2.0 train and got similar performance in eval. I'd appreciate if someone can help me understand why this is the case. Thanks!
```

model:
  arch: blip_vqa
  model_type: vqav2
  image_size: 480

datasets:
  coco_vqa: # name of the dataset builder
    type: eval
    vis_processor:
        eval:
          name: "blip_image_eval"
          image_size: 480
    text_processor:
        eval:
          name: "blip_question"

run:
  task: vqa
  # optimization-specific
  batch_size_train: 16
  batch_size_eval: 32
  num_workers: 4

  # inference-specific
  max_len: 10
  min_len: 1
  num_beams: 3
  num_ans_candidates: 128
  inference_method: "rank"

  seed: 42
  output_dir: "output/BLIP/VQA"

  evaluate: True
  test_splits: ["val"]

  # distribution-specific
  device: "cuda"
  world_size: 8
  dist_url: "env://"
  distributed: True
```

2023-08-14 22:54:50,117 [INFO] Overall Accuracy is: 83.42

2023-08-14 22:54:50,117 [INFO] Per Answer Type Accuracy is the following:
2023-08-14 22:54:50,117 [INFO] other : 75.79
2023-08-14 22:54:50,118 [INFO] yes/no : 97.77
2023-08-14 22:54:50,118 [INFO] number : 71.00

**Comment by dxli94** [comment]
@essamsleiman The result you got is on the validation set. Results in the papers are on test-dev / test-std.

You may also find this helpful: https://github.com/salesforce/LAVIS/issues/423.

**Comment by Pefect96** [comment]
I have the same problem as you. Have you got the final result? How to use the evaluation platform.
